include/boost/geometry/strategies/geographic/side.hpp

```diff
@@ -120,6 +122,11 @@ class geographic
         typedef typename FormulaPolicy::template inverse
                     <calc_t, false, true, false, false, false> inverse_formula;
 
+        if (geometry::equals(p, p1) || geometry::equals(p, p2))
```

Comment:
`equals` needs geographic/spherical point_in_point strategy.

Reply:
Btw, is it a fix for a bug?

Reply:
>equals needs geographic/spherical point_in_point strategy.

right

>Btw, is it a fix for a bug?

Indeed, this fixes this bug https://github.com/boostorg/geometry/issues/765.